src/main/jte/index.jte

```diff
@@ -2,10 +2,13 @@
 @import edu.kit.hci.soli.domain.Room
 @import edu.kit.hci.soli.template.JteContext
 @import edu.kit.hci.soli.template.PageSpec
+@import edu.kit.hci.soli.domain.Booking
+@import java.util.List
 
 @param JteContext context
 @param LoginStateModel login
 @param Room room
+@param List<Booking> bookings
```

Comment:
parameter not used

Reply:
It seems like the changes to this file can just be reverted, so that's probably the easiest option